.dialyzer_ignore.exs

```diff
@@ -1,4 +1,6 @@
 [
+  ~r/lib\/mongoose_push\/metrics\/telemetry_metrics\.ex.*child_spec.*no local return/,
```

Comment:
It would be nice to put a comment here why do we include those ignores. Maybe link to a bug report?